lucille-examples/lucille-distributed-example/kafka/server.properties

```diff
@@ -0,0 +1,138 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
would be nice to comment anything here that's different from a boilerplate kafka server.properties
listeners and advertised.listeners was changed, right? -- anything else?